src/test/suite/fixtures/recorded/languages/scss/chuckArgue4.yml

```diff
@@ -0,0 +1,29 @@
+languageId: scss
```

Comment:
Probably worth supporting "name" from cursor anywhere inside `$amount: 100%` to refer to `$amount`.  Also, not sure if you support this one, but cursor anywhere inside `$amount: 100%`, saying "value" should refer to `100%`